bot/exts/evergreen/emoji_count.py

```diff
@@ -80,10 +85,11 @@ async def ec(self, ctx: commands.Context, *, emoji: str = None) -> Optional[str]
         emoji_dict = self.emoji_list(ctx, emoji_dict)
```

Comment:
Can't this be done just above this? If you use a defaultdict you dont need to add a list to the dict at all, it will create one if it doesnt exist
```py
emoji_dict = defaultdict(list)

for emoji in emojis:
    emoji_dict[emoji_prefix].append(emoji)
```
etc